required.mk

```diff
@@ -22,8 +22,12 @@ PYTHON ?= $(shell which python3)
 RSCRIPT ?= $(shell which Rscript)
 
 # Jekyll version on GitHub
-# Sync with https://pages.github.com/versions/
-GITHUB_JEKYLL_VERSION = 3.7.4
+ifneq (, $(wildcard ./Gemfile.lock))
+  GITHUB_JEKYLL_VERSION := $(shell sed -n '/jekyll\ (=.*)/s|.*(= \(.*\))|\1|p' Gemfile.lock)
```

Comment:
Here, I'm using the line in `Gemfile.lock` that reads
```
      jekyll (= 3.7.4)
```
There are other lines where jekyll is listed. I use the one above because it is a direct descendant of `github-pages` gem (Line 36 in Gemfile.lock)